### PR TITLE
Update Alerts section

### PR DESCRIPTION
**Pull request by cpipilas**
Updates in Alerts:
- Overhaul of Alerts Events page
- Introduce new Alert Rules page
- Update Alerts Overview
- Restructure Alerts left nav
- Add screenshots and links to other docs
- Minor update in Alerts Guide

### CODE REVIEW

File: docs/alerts/alert-events.md [hunk 1]
```diff
@@ -1,21 +1,105 @@
...
+
+In the alert event detail view, a highly visible “Go to Troubleshooting” link is available. This link allows you to dive straight into the core of any issue. Clicking this link redirects you to the most relevant App report, automatically filtered by the alert trigger time and associated tags, such as hostname. This feature improves troubleshooting efficiency by directing you to the exact point of interest based on the alert details.
+
+In the same section, you may also find the last log events that occurred immediately before the alert was triggered, offering immediate and relevant context for quick troubleshooting.
+
+![Alert Event Troubleshooting](../images/alerts/alert-event-troubleshooting.png)
+
+### Built-in Correlation for Troubleshooting
+
+The next sections provide the means to correlate alert events with other relevant information from Sematext.
```
Comment (otisg) [inline docs/alerts/alert-events.md:76]: Maybe "....with other relevant observability data"?

File: docs/alerts/alert-events.md [hunk 2]
```diff
@@ -1,21 +1,105 @@
...
+
+Below the Sematext Events, a chart provides a stacked view of alert events from the current App and any [Connected Apps](https://sematext.com/docs/guide/connected-apps/). This chart visually represents how multiple alerts are interacting over time, helping you understand if other Apps are experiencing similar issues.
+
+![Alert Events Chart From Connected Apps](../images/alerts/alert-event-events-chart.png)
+
+##### Alert Events Table
+
+A table follows the stacked view chart, listing alert events for both the current App and any connected Apps. Each row displays key details such as the alert type, alert rule name, triggered time, and priority. Two actions are available on each row:
+
+- A split screen icon that opens the [Split Screen](https://sematext.com/docs/guide/split-screen/) feature, allowing you to compare side-by-side reports from the current alert's App with those from the selected alert's App. This way you can easily identify relationships between different Apps, helping determine if a broader issue is affecting multiple components of your system.
```
Comment (otisg) [inline docs/alerts/alert-events.md:94]: A split screen icon that opens ==> The split screen icon opens

File: docs/alerts/alert-rules.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
+title: Alert Rules
+description: Alert rules are essential for monitoring your applications and infrastructure. They allow you to set specific conditions under which you will be notified of changes in metrics or events
```
Comment (otisg) [inline docs/alerts/alert-rules.md:2]: + logs...

File: docs/alerts/alert-rules.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+![Alert Rules](../images/alerts/alert-rules.png)
+
+Fields available:
+
+- **Priority:** Specifies the importance of the alert rule.
+- **Integrations:** Lists the [notification methods](https://sematext.com/docs/alerts/alert-notifications/) (e.g., Email, Slack, Webhook) tied to the alert.
+- **Alert Rule Name:** The name of the alert rule.
+- **App:** The App to which this alert rule is applied.
+- **Last Notification:** Shows the time of the last alert notification. Clicking on this leads to the specific [alert event](https://sematext.com/docs/alerts/alert-events/) details.
```
Comment (otisg) [inline docs/alerts/alert-rules.md:18]: It's probably actually the last time the AR was triggered. One can turn off notifications.

File: docs/alerts/alert-rules.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+- **App State:** Indicates whether the app is receiving data.
```
Comment (otisg) [inline docs/alerts/alert-rules.md:19]: app ==> App
is receiving data ==> is currently receiving data ?

File: docs/alerts/alert-rules.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+The alert rule summary gives you a quick overview of the rule’s key configuration details.
+
+![Alert Rule Summary](../images/alerts/alert-rule-summary.png)
+
+Fields available:
+
+- **App:** The App to which this alert rule is applied.
+- **Threshold:** The condition that triggers the alert (e.g., CPU usage > 80%).
+- **Interval:** The time interval for which the threshold must be crossed before triggering the alert.
+- **Notifications:** Whether notifications for this rule are enabled or disabled.
```
Comment (otisg) [inline docs/alerts/alert-rules.md:43]: Maybe worth saying that one will still see alert events in the SC UI even if notifications have been disabled

File: docs/alerts/alert-rules.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+The table includes:
+
+- **Alert Time:** The exact time when the alert was triggered. Clicking on this leads to the specific alert event details.
+- **Tags used for Grouping:** A list of tags by which the alert metric is grouped, such as `os.host`.
+
+This section helps you track historical alert events.
+
+### Rule Details
+
+The **Rule Details** section provides an quick view of the alert rule’s configuration.
```
Comment (otisg) [inline docs/alerts/alert-rules.md:71]: an quick ==> a quick

File: docs/alerts/alert-rules.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+![Alert Rule Details](../images/alerts/alert-rule-details.png)
+
+Fields included:
+
+- **Metrics:** The specific metric(s) being monitored.
+- **Priority:** The priority of the alert rule
```
Comment (otisg) [inline docs/alerts/alert-rules.md:78]: . at the end

File: docs/alerts/alert-rules.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+
+3. **From the Alerts Page:** Access the Alerts page through the left menu panel and select **'+ New Alert Rule'** from the dropdown menu located at the top right of the screen.
+
+![Create Alert Rules](../images/alerts/create-alert-rules.gif)
+
+Once the **New Alert Rule** window opens, you can edit the rule name and select one of the available options (tabs) on the left to configure the alert, as detailed in the following subsections.
+
+### Alert Metric
+
+If you open the **New Alert Rule** window from a specific App, that App will be automatically selected in the dialog. Next to the App selection, a dropdown allows you to choose one or multiple metrics to receive alerts for.
```
Comment (otisg) [inline docs/alerts/alert-rules.md:104]: Wondering how we can make it clear this is all.... maybe the whole page really... all about Monitoring/Logs, and not e.g. Synth.

File: docs/alerts/alert-rules.md [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+
+Historical data is analyzed to determine if an alert fits the regular stream of spikes and dips in metrics. If it does, notifications will not be sent. The filter combination is used to identify previous spikes and dips, grouping them into similar values (±5% assumed to be the same). The least common multiple (LCM) in these groups is calculated to check if the current value fits any of them.
+
+### Meta Data
+
+The **Meta Data** tab allows you to enter details about the alert, such as the alert name, description, and runbook.
+
+#### Runbook
+
+The runbook serves as a detailed guide outlining procedures, recommended actions, and best practices to follow when an alert is triggered. It ensures a standardized and efficient approach to managing routine operations, handling system updates, addressing issues, and promptly restoring services upon alert activation.
```
Comment (otisg) [inline docs/alerts/alert-rules.md:142]: We have a pretty solid Glossary page/guide for Runbooks.... if we want to link to that?

File: docs/alerts/index.md [hunk 11]
```diff
@@ -7,105 +7,54 @@ frameborder="0" allow="autoplay; encrypted-media"
...
 
-<br/>
-
 ## Alert Sources
+
 Alerts can operate on three different **sources** of data:  
 
-1. **Metrics** alerts work with infrastructure metrics
-1. **Logs** alerts are based on various types of logs
-1. **Experience** alerts are concerned with *real user monitoring* data - RUM
```
Comment (otisg) [inline docs/alerts/index.md:38]: Btw this 1. 1. 1. automatically turns that into a numbered list.

**Review by otisg — CHANGES_REQUESTED** [review]
LGTM overall.

File: docs/alerts/index.md (hunk 11 above)
Comment (cpipilas) [reply docs/alerts/index.md:38]: GitHub Markdown is quite different from MkDocs, so I'm trying to play it safe

File: docs/alerts/alert-rules.md (hunk 3 above)
Comment (cpipilas) [reply docs/alerts/alert-rules.md]: I just added log events, if this is what you had in mind.

File: docs/alerts/alert-rules.md (hunk 9 above)
Comment (cpipilas) [reply docs/alerts/alert-rules.md:104]: We partially address this in the last section, **Additional Resources on Alert Rules**, where we provide a specific link for creating Synthetics alerts.

File: docs/alerts/index.md [hunk 12]
```diff
@@ -7,105 +7,54 @@ frameborder="0" allow="autoplay; encrypted-media"
...
 
-Alert type | Metrics | Logs | Experience
---- | --- | --- | ---
-Threshold | yes | yes | yes
-Anomaly | yes | yes | yes
-Heartbeat | yes | no | no
+Alert type | Metrics | Logs | Experience | Synthetics
+--- | --- | --- | --- | ---
+Threshold | yes | yes | yes | yes
+Anomaly | yes | yes | yes | no
```
Comment (fulyauluturk) [inline docs/alerts/index.md:45]: @cpipilas it is also possible to create Anomaly ARs with Synthetics metrics

**Pull request merged**